packages/protobuf/scripts/protobuf-patches/TxOutputType.js

```diff
@@ -25,10 +25,20 @@ export type TxOutputType =
           orig_index?: number,
           payment_req_index?: number,
       }
+      | {
+        address: string,
+        address_n?: typeof undefined,
+        script_type?: ChangeOutputScriptType,
+        amount: UintType,
+        multisig?: MultisigRedeemScriptType,
+        orig_hash?: string,
+        orig_index?: number,
+        payment_req_index?: number,
+    }
```

Comment:
used like this in tests: https://github.com/trezor/trezor-suite/blob/782c8aa5e7f21781ab51cff197d139e4e1b1a4c4/packages/connect/e2e/__fixtures__/signTransactionReplace.ts#L115

Reply:
@martykan please try to change https://github.com/trezor/trezor-suite/blob/782c8aa5e7f21781ab51cff197d139e4e1b1a4c4/packages/connect/e2e/__fixtures__/signTransactionReplace.ts#L117 to `PAYTOADDRESS`. it is possibly wrong in the test. and with this change, this type loosening here should not be needed anymore, right?